sample-app/src/components/Dropdown.tsx

```diff
@@ -23,22 +26,41 @@ interface Props {
 export default function Dropdown({
   options,
   onClickOption = () => {},
+  focusedSectionIndex,
   focusedOptionIndex,
   optionIdPrefix,
   cssClasses
 }: Props): JSX.Element | null {
-  function renderOption(option: Option, index: number) {
+  function renderSection(section: {results: Option[], label?: string}, sectionIndex: number) {
```

Comment:
nit: spacing inside the curly braces. hmm I thought `"object-curly-spacing": ["error", "always"]` would catch this. Can you double check if that eslint rule is working?

Reply:
Added the spaces, but seems like the eslint rule isn't working because it didn't give any error when there were no spaces

Reply:
Can you do a brief check on why that is? it seems to work for the top level src folder outside of sample-app

Reply:
I looked into it and seems like the sample-app folder is one of the ignoredPatterns in the eslint config. Should I remove it from there so that it gives the errors?

Reply:
inside package.json there's an eslintConfig part for sample-app that's different from the top level eslint. So I believe the ignorePatterns stuff shouldn't override the internal rules for sample-app

Reply:
I didn't change anything, so not sure why, but now I'm getting the eslint errors in the sample-app